Makefile

```diff
@@ -148,6 +148,20 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: kustomize ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | $(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f -
 
+# Build and upload docker image to the local Kind cluster
+.PHONY: docker-load
+docker-load: docker-build
+	kind load docker-image ${IMG} --name etcd-operator-kind
```

Comment:
I believe it's unnecessary to use a fixed name like `etcd-operator-kind`. Instead, using the default name or allowing configuration through an environment variable could be more flexible. Note that the latter option would necessitate an extra setup step for kind creation.

Reply:
I used `name` from [HOW-TO-START-DEVELOPMENT.md](https://github.com/aenix-io/etcd-operator/blob/main/HOW-TO-START-DEVELOPMENT.md). Do you think I should pass it through a Makefile env?

Reply:
Yep, I suppose it should be like that.

Honestly, I wanted to make documentation [about development process](https://github.com/aenix-io/etcd-operator/blob/main/HOW-TO-START-DEVELOPMENT.md) smaller, just put steps for creating `kind` cluster with all required installation as a sequence of Makefile runs. You can skip this comment, I hope I'll have time to do it separately.